manifest.yml

```diff
@@ -2,7 +2,8 @@
 applications:
 - name: cloudcache-pizza-store
   path: target/PCC-Sample-App-PizzaStore-1.0.0-SNAPSHOT.jar
-  buildpack: java_buildpack_offline
+  buildpacks:
+    - https://github.com/cloudfoundry/java-buildpack.git
```

Comment:
For pizza app, some customers are not able to connect to github because of their security policies. I don't think it's a big deal, but we might wan to double check with @srikanthmanvi about this.

Reply:
This change was from the original work to make the app work with service gateways. I do  not recall the history behind the change, but I'll review those earlier commits.

Reply:
I believe the offline buildpack was not being actively maintained? By pulling it from github we get a current version. Offline versions of the buildpack can be built following instructions in https://github.com/cloudfoundry/java-buildpack/blob/main/README.md.

Reply:
Sarah Abbey's team is maintaining the offline buildpack i think. I suppose this is fine. If anything breaks downstream it's an easy fix to make.